oauth2_http/javatests/com/google/auth/oauth2/ExternalAccountCredentialsTest.java

```diff
@@ -347,6 +392,59 @@ public void fromJson_pluggableAuthCredentialsWithServiceAccountImpersonationOpti
     assertNotNull(credential.getCredentialSource());
     assertEquals(2800, credential.getServiceAccountImpersonationOptions().getLifetime());
 
+    PluggableAuthCredentialSource source =
+        (PluggableAuthCredentialSource) credential.getCredentialSource();
+    assertEquals("command", source.getCommand());
+    assertEquals(30000, source.getTimeoutMs()); // Default timeout is 30s.
+    assertNull(source.getOutputFilePath());
+    assertNull(credential.getUniverseDomain());
+  }
+
+  @Test
+  public void fromJson_pluggableAuthCredentials_withUniverseDomain() {
```

Comment:
This test should be named withoutUniverseDomain or should just be deleted since it looks like the test above already covers the no universe domain scenario.